### PR TITLE
Documentation: meson build description updated.

### PR DESCRIPTION
**Pull request by myungjoo**
Per the request of #3320, the document is updated.

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

CC: @shortcipher3

### CODE REVIEW

File: Documentation/getting-started-meson-build.md [hunk 1]
```diff
@@ -83,7 +93,12 @@ $ ninja -C build
 
 Assuming you have configured meson at ```./build```.
 ```bash
-$ ninja -C build install
+$ sudo ninja -C build install  # if the destination is not user directory
+```
+
+Of if you want somewhere else:
```
Comment (gichan-jang) [inline Documentation/getting-started-meson-build.md:99]: Or?

Comment (myungjoo) [reply Documentation/getting-started-meson-build.md]: Corrected.

File: Documentation/getting-started-meson-build.md [hunk 2]
```diff
@@ -19,41 +22,48 @@ The following dependencies are needed to compile/build/run.
 * glib 2.0
 * meson >= 0.50
 * ninja-build
-* Neural network frameworks or libraries for plugins (e.g., tensorflow) you want to use, including their pkgconfig files or mechanisms to support meson to discover the corresponding frameworks. If you use our packages for tensorflow/pytorch, you do not need to worry.
-    * Possible frameworks for "extra" plugins: tensorflow, pytorch, protobuf, flatbuf, openvino, ncsdk2, Verisilicon-Vivante, SNPE, ...
+* Neural network frameworks or libraries for plugins (e.g., tensorflow) you want to use, including their pkgconfig files or mechanisms to allow meson to discover its headers and libraries. If you use development pacakges packaged by us for tensorflow/pytorch/openvino/..., you do not need to worry.
```
Comment (wooksong) [inline Documentation/getting-started-meson-build.md:25]:
```suggestion
* Neural network frameworks or libraries for plugins (e.g., tensorflow) you want to use, including their pkgconfig files or mechanisms to allow meson to discover its headers and libraries. If you use development packages packaged by us for tensorflow/pytorch/openvino/..., you do not need to worry.
```

File: Documentation/getting-started-meson-build.md [hunk 3]
```diff
@@ -19,41 +22,48 @@ The following dependencies are needed to compile/build/run.
...
 ```bash
-$ sudo apt install meson ninja-build gcc g++ libglib2.0-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev
+$ sudo apt-get install meson ninja-build gcc g++ libglib2.0-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev
+```
+
+Optional utilities:
+```bash
+$ sudo add-apt-repository ppa:nnstreamer/ppa
+$ sudo apt-get update
+$ sudo apt-get install libiniparser-dev=4.1 # if your inparser is < 4.1
```
Comment (kimjh12) [inline Documentation/getting-started-meson-build.md:39]:
```suggestion
$ sudo apt-get install libiniparser-dev=4.1 # if your iniparser is < 4.1
```

File: Documentation/getting-started-meson-build.md [hunk 4]
```diff
@@ -19,41 +22,48 @@ The following dependencies are needed to compile/build/run.
...
-ARCH := $(shell if [[ $(shell uname -m) =~ i[345678]86 ]]; then echo x86_32; else echo $(shell uname -m); fi)
+*Option A*: You may install optional developmental packages from nnstreamer PPA for more subplugins and extra plugins (choose what you need):
 ```
-
-To:
-```makefile
-ARCH := $(shell uname -m | sed -e 's/i[3-8]86/x86_32/')
+$ sudo add-apt-repository ppa:nnstreamer/ppa
+$ sudo apt-get update
+$ sudo apt-get install libedgetpu-dev libflatbuffers-dev libgrpc-dev openvino-dev libpaho-mqtt-dev libprotobuf-dev pytorch tensorflow-c-dev tensorflow-lite-dev tensorflow2-lite-dev tvm-runtime
```
Comment (jaeyun-jung) [inline Documentation/getting-started-meson-build.md:57]: tvm-runtime > tvm-runtime-dev to build tvm subplugin

File: Documentation/getting-started-meson-build.md (hunk 2 above)
Comment (myungjoo) [reply Documentation/getting-started-meson-build.md:25]: Applied.

File: Documentation/getting-started-meson-build.md (hunk 3 above)
Comment (myungjoo) [reply Documentation/getting-started-meson-build.md:39]: Applied

File: Documentation/getting-started-meson-build.md (hunk 4 above)
Comment (myungjoo) [reply Documentation/getting-started-meson-build.md:57]: Applied.

**Review by jaeyun-jung — APPROVED** [review]

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**